### ISSUE TITLE
Please include formatCardCVC

### ISSUE THREAD

**Issue by 0xStarcat** [faaez/creditcardutils#2]
Hi, the library seems to not include the function `formatCardCVC`. Any plans on supporting this formatting method?

**Comment by faaez** [comment]
Yes, could add that in the next few days. Or feel free to create a PR if you like.
